recipes/MOABB/MotorImagery_decoding/BNCI2014001/train.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/python
+"""
+Recipe for training a compact CNN to decode motor imagery from single EEG trials.
```

Comment:
It looks like the train.py is "architecture-agnostic". I suggest removing the CNN mention here and add it in the header of the yaml file instead.